ArtCeil.html

```diff
@@ -0,0 +1,230 @@
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="Styles/Style.css">
+  <title>Потолочные покрытия</title>
+</head>
+
+<body>
+  <header class="header">
+    <div class="header__top">
+      <div class="header__top top__menu">
+        <div class="header__orangeline"></div>
+        <div class="header__greenline"></div>
+        <div class="header__blueline"></div>
+        <div class="header__menu">МЕНЮ</div>
+      </div>
+      <div class="header__contacts">
+        <div>Email:<span class="header__contacts_content">shiningfinger@list.ru</span></div>
+        <div>Телефон:<span class="header__contacts_content">+7 (985) 905-12-51</span></div>
+      </div>
+      <div class="header__logo">
+        <img class="header__logo-image" src="../Artceil/Images/logo.png">
+        <div class="header__logo_content">Art<span class="Ceil">Ceil</span><span class="tochka">.</span>
+          <div class="header__logo_name">Цитадель потолочных покрытий</div>
+        </div>
+      </div>
+      <div class="header__search">
+        <input class="header__search_content" placeholder="Поиск">
+        <img src="../Artceil/Images/search.png" class="header__search_image">
+      </div>
+      <div class="header__connection">
+        <div class="header__connection_calling"><button class="header__button button_calling">Обратный вызов</button>
+          <img class="picture_calling" src="../Artceil/Images/callback.png">
+        </div>
+        <div class="header__connection_calling"><button class="header__button button_favorite">Оформить заказ</button>
+          <img class="picture_favorite" src="../Artceil/Images/favorite.png">
+        </div>
+      </div>
+    </div>
+  </header>
+  <main>
+    <div class="header__bot container">
+      <div class="header__bot_menu"><a class="brends">Бренды</a>
+        <p class="">,&nbsp</p> <a class="brends"> Категории</a>
+      </div>
+      <div class="header__logo_name name__main">Цитадель потолочных покрытий</div>
+      <div class="header__maintext">
+        <p class="header__maintextt">ВЫСТАВОЧНЫЙ ЗАЛ</p>
+      </div>
+      <div class="main_logo">
+        <img src="./Images/logo_2.png">
+      </div>
+    </div>
+    <div class="main-brends container">
+      <h1>БРЕНДЫ</h1>
+      <div class="brends-contant">
+        <img src="./Images/Brends_picture_1.png">
+        <img src="./Images/Brends_picture_2.png">
+        <img class="brends-contant__picture" src="./Images/Brends_picture_3.png">
+        <div>
+          <img class="brends-contant__picture" src="./Images/Brends_picture_4.png">
+          <div class="brends-contant__title">HUNTER DUGLAS</div>
+        </div>
+        <img class="brends-contant__picture" src="./Images/Brends_picture_5.png">
+      </div>
+      <div>
+        <section class="main-category">
+          <h1>КАТЕГОРИИ</h1>
+          <section class="category-contant">
+            <h2 class="category-texture__text">TEКСТУРА</h2>
+            <div class="texture-picture">
```

Comment:
конкретные названия категории не в тему тут)
 в этом блоке список превью категорий
предлагаю так назвать

```suggestion
            <div class="main-content-categories-list">
```